src/manager/msNotificationManager.ts

```diff
@@ -27,47 +32,72 @@ export class MsNotificationManager extends MsManager {
             ? new Date(Me.stateManager.getState('notification-check'))
             : new Date();
 
-        const message = new Soup.Message({
-            method: 'GET',
-            uri: new Soup.URI(
-                `${API_SERVER}/notifications?lastCheck=${previousCheck.toISOString()}`
-            ),
-        });
+        const message = Soup.Message.new(
```

Comment:
Why do they use these weird constructors, I wonder...

Reply:
it's the only constructor that I found that is both compatible with soup 2.4 and soup 3